vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/LruCache.java

```diff
@@ -0,0 +1,32 @@
+package io.vertx.sqlclient.impl;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * A reusable cache implementation based on LRU(Least Recently Used) strategy.
+ */
+public class LruCache<K, V> extends LinkedHashMap<K, V> {
```

Comment:
make this package private

Reply:
I think it's actually not need to have this intermediary class, just make the cache class extends `LinkedHashMap`